src/java_bytecode/library/converter.cpp

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************\
+
+Module:
+
+Author: Dario Cattaruzza
+
+Date: November 2017
+
+\*******************************************************************/
+
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <vector>
+#include <cstring>
+
+/// Loads a binary file into a c array and creates c and h files to
+/// access  it from code.
+int main(int argc, char *argv[])
+{
+  unsigned size;
+  const char *varname = 0;
+
+  if(argc<2 || strlen(argv[1])==0)
+  {
+    std::cout << "Usage: converter VARNAME\n";
+    std::cout << "\n";
+    return 1;
+  }
+  varname=argv[1];
+
+  printf("// File automatically generated by " __FILE__ "\n\n");
+
+  size=0;
```

Comment:
As this is C++ code, please move the declaration to the point of definition, and please make this a `std::size_t` to avoid overflow.